python/ml_croissant/ml_croissant/_src/structure_graph/graph.py

```diff
@@ -86,15 +86,27 @@ def from_json_to_rdf(data: Json) -> rdflib.Graph:
     return graph
 
 
+def _append_to_list(dict: dict[str, list[Any]], key: str, value: Any):
```

Comment:
You may be able to use a defaultdict instead: https://docs.python.org/3/library/collections.html#collections.defaultdict

```py
d = defaultdict(list)
d[key].extend([1,2,3])
```

Reply:
Done.